addons/point_of_sale/static/src/js/models.js

```diff
@@ -1831,6 +1836,8 @@ exports.Orderline = Backbone.Model.extend({
         var full_name = this.product.display_name;
         if (this.description) {
             full_name += ` (${this.description})`;
+        } else if (this.full_product_name) {
+            full_name = this.full_product_name
```

Comment:
Not sure about this. If I understand correctly, `full_product_name` already contains `display_name + description`, right? Wouldn't it be equivalent to do:
```js
    get_full_product_name: function () {
        if (this.full_product_name) {
            return this.full_product_name;
        }
        var full_name = this.product.display_name;
        if (this.description) {
            full_name += ` (${this.description})`;
        }
        return full_name;
    },
```

Reply:
Yes, `full_product_name` contains `display_name + description`.

I just wanted to use it as fallback solution but in the end, it is the same.